### PR TITLE
Export divergent changes

### PR DESCRIPTION
**Pull request by bdw-g**
These were changes that had diverged locally which now have to be exported again.

### CODE REVIEW

File: dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/JsonResponseFileTest.java [hunk 1]
```diff
@@ -30,12 +31,12 @@
 public class JsonResponseFileTest {
 
     // replace single quotes to double quotes to avoid \" messup.
-    private void check(String json, String... exp) throws JsonProcessingException {
+  private void check(String json, String... exp) throws IOException, JsonProcessingException {
```
Comment (shevek-google) [inline dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/JsonResponseFileTest.java:34]: formatting

File: dumper/lib-dumper-spi/src/main/java/com/google/edwmigration/dumper/plugin/lib/dumper/spi/TeradataLogsDumpFormat.java [hunk 2]
```diff
@@ -180,6 +180,7 @@ enum HeaderForAssessment {
     }
 
     enum HeaderLSql {
+        QueryID,
```
Comment (shevek-google) [inline dumper/lib-dumper-spi/src/main/java/com/google/edwmigration/dumper/plugin/lib/dumper/spi/TeradataLogsDumpFormat.java:183]: The connector uses this to drive the underlying select. Do we know why this got out of sync? Is someone trying to read this column, which the connector isn't currently writing?

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/JsonResponseFile.java [hunk 3]
```diff
@@ -85,7 +85,8 @@ private static List<? extends String> to_arguments(@Nonnull File file) throws IO
     }
 
     @VisibleForTesting
-    /* pp */ static List<? extends String> to_arguments(@Nonnull String text) throws JsonProcessingException {
+    /* pp */ static List<? extends String> to_arguments(@Nonnull String text)
+        throws IOException, JsonProcessingException {
```
Comment (shevek-google) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/JsonResponseFile.java:89]: formatting: either indent, or rejoin?

File: dumper/lib-dumper-spi/src/main/java/com/google/edwmigration/dumper/plugin/lib/dumper/spi/TeradataLogsDumpFormat.java (hunk 2 above)
Comment (bdw-g) [reply dumper/lib-dumper-spi/src/main/java/com/google/edwmigration/dumper/plugin/lib/dumper/spi/TeradataLogsDumpFormat.java:183]: This field is used by [`TeradataLogDumpCsvTranspilerIngester`](https://source.corp.google.com/piper///depot/google3/java/com/google/cloud/helix/edwmigration/translation/engine/lang/teradata/transpiler/TeradataLogDumpCsvTranspilerIngester.java;rcl=478391678;l=90) and I presume that isn't recent; [`TeradataLogsConnector`](https://github.com/google/dwh-migration-tools/blob/849e0849f5e65eb78141f19d756f88723813ccf4/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java#L98) writes this field.

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/JsonResponseFile.java (hunk 3 above)
Comment (bdw-g) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/JsonResponseFile.java:89]: done

File: dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/JsonResponseFileTest.java (hunk 1 above)
Comment (bdw-g) [reply dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/JsonResponseFileTest.java:34]: In Jackson 2.9.10, `IOException` is thrown by [`ObjectMapper.readTree(String)`](https://source.corp.google.com/piper///depot/google3/third_party/java_src/jackson2/v2_9_10/jackson-databind-2.9.10.8/com/fasterxml/jackson/databind/ObjectMapper.java;bpv=1;bpt=1;rcl=478391678;l=2551?q=ObjectMapper.readTree&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental&gsn=readTree&gs=kythe%3A%2F%2Fgoogle3%3Flang%3Djava%3Fpath%3Dcom.fasterxml.jackson.databind.ObjectMapper%23ea6fe17ad8e74d2217ccfb651bd03123d02774db95d64767e405d380326fe4b3), which is why this exception had to be added.

**Review by paolomorandini — APPROVED** [review]

**Review by shevek-google — APPROVED** [review]

**Pull request merged**